docs/howto-build-chromium-riscv64.md

```diff
@@ -0,0 +1,149 @@
+**How to build Chromium RISCV64**
+
+<!-- TOC -->
+
+- [1. Hardware environment](#1-hardware-environment)
+- [2. Install depot\_tools](#2-install-depot_tools)
+- [3. Get the code](#3-get-the-code)
+- [4. Install additional build dependencies](#4-install-additional-build-dependencies)
+- [5. Checkout the branch](#5-checkout-the-branch)
+- [6. Run the hooks](#6-run-the-hooks)
+- [7. Build Rust toolchain](#7-build-rust-toolchain)
+- [8. Setting up the build](#8-setting-up-the-build)
+- [9. Build Chromium](#9-build-chromium)
+
+<!-- /TOC -->
+
+# 1. Hardware environment
+
+All operations in this article have been verified under the following system environment:
+
+```
+$ lsb_release -a
+No LSB modules are available.
+Distributor ID: Ubuntu
+Description:    Ubuntu 20.04.5 LTS
+Release:        20.04
+Codename:       focal
+```
+
+# 2. Install depot\_tools
+
+Clone the `depot_tools` repository:
+
+```shell
+git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
+```
+
+Add `depot_tools` to the end of your PATH (you will probably want to put this
+in your `~/.bashrc` or `~/.zshrc`). Assuming you cloned `depot_tools`
+to `/path/to/depot_tools`:
+
+```shell
+export PATH="$PATH:/path/to/depot_tools"
+```
+
+# 3. Get the code
+
+Create a `toolchain` directory for the special Android RISCV64 toolchain to substitute
+Chromium official toolchain.
+```shell
+mkdir ~/toolchain && cd ~/toolchain
+git clone --branch riscv-ndk-release-r23 https://github.com/riscv-android-src/platform-prebuilts-clang-host-linux-x86.git
```

Comment:
wonder if you write the wrong branch info for clang and ndk repos?

Reply:
I double checked it and the branch info is correct.